R/idr0021 - Segmentation.ipynb

```diff
@@ -0,0 +1,702 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Image Segmentation using the idr0021 dataset\n",
+    "\n",
+    "This notebook uses the idr0021 dataset (original: [idr0021-lawo-pericentriolarmaterial/experimentA](http://idr.openmicroscopy.org/webclient/?show=project-51)) and tries to reproduce some of the analysis published in ['Subdiffraction imaging of centrosomes reveals higher-order organizational features of pericentriolar material'](https://doi.org/10.1038/ncb2591). In particular we're trying to get a figure similar to [Figure 1](https://www.nature.com/articles/ncb2591/figures/1) of the article. \n",
+    "\n",
+    "Each user has a partial copy of the dataset ('Lab1' / YOUR USER / Project: 'idr0021'). Some of the images are also part of the dataset 'R-dataset' which you can use for practicing the segmentation and annotation steps."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Exercise:** Go to https://outreach.openmicroscopy.org , log in as your given user, and find the dataset ('Lab1' / YOUR USER / Project: 'R-dataset'). Leave the window open, as you will use OMERO.web to check the results as we're going along."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## How to use this notebook"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The grey blocks with the In\\[ \\] notation to the left are code blocks. Click on a block to select it and click the play button in the toolbar to execute the block of code (alternatively hold SHIFT key and hit RETURN). The notation will change to In\\[*\\] to indicate that this code is currently running. The output is displayed below the block. Wait for it to finish. When the execution is finished it will change to ln\\[1\\] showing that this was the first block of code you ran (ln\\[2\\], ln\\[3\\], ... for the next executed code blocks). \n",
+    "\n",
```

Comment:
RETURN -> ENTER

Reply:
``The notation will change...`` sounds strange. 
Maybe: ``The star in the square brackets to the left of the cell will turn into a number.``

Reply:
For some reason, what should be ``[*]`` here, resolves as ``ln[*]`` in the notebook. What does the ``ln`` mean ? I think it should be removed.